platform-sdk/docs/hashgraph_glossary.md

```diff
@@ -0,0 +1,250 @@
+# Hashgraph Glossary
+
+<a name="top"/>
+
+- [Rounds](#rounds)
+- [Fields of an event](#fields-of-an-event)
+- [Events](#events)
+- [Event relationships](#event-relationships)
+- [Generations](#generations)
+- [Address Books](#addressbooks)
+- [Parameters](#parameters)
+
+<a name="rounds"/>
+
+## Rounds
+
+- __Pending__ round - the round number of the earliest round that hasn't yet reached consensus. This is the round whose
+  judges are currently being determined. Those consensus calculations use the roster whose roster round is the current
+  `pendingRound`. Any new event is created containing a `birthRound` field equal to the current `pendingRound`. All
+  events currently in the hashgraph should have birth rounds in the range from `pendingRound - roundsAncient` to
```

Comment:
The code currently calls this `roundsNonAncient`, which to me is more clear than this name. However, the code also uses `roundsExpired` which should be `roundsNonExpired` because it is used in the same manner but for a different concept as `roundsNonAncient`. I suggest we make both include "non" because it means the number of rounds that are NOT ancient and NOT expired.

Reply:
I agree

Reply:
the `roundsExpired` is really a sync gossip concept, so maybe it should not be here. if we move to a new gossip algorithm, this might not exist anymore

Reply:
Also, this paragraph talks about `pending`, `roster`, `birth`, and `ancient` rounds all at once.
1. I suggest to have a separate paragraph for each of these.
2. The paragraphs that define these rounds should go in a logical order, so that they don't have forward-references to terms defined below (as much as it's possible, of course.)